Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -2633,7 +2633,6 @@ def main():
         connection_args['username'] = username
         connection_args['password'] = password
         connection_args['autologin'] = True
```

Comment:
```suggestion
        connection_args['autologin'] = True
        connection_args['basic'] = True
```
Please do not remove basic. this issue was handled in a better way in https://github.com/demisto/content/pull/20662